pkg/ptr/ptr_test.go

```diff
@@ -0,0 +1,57 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+
+package ptr
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"testing"
+)
+
+func TestSlice(t *testing.T) {
```

Comment:
That would be really hard to achieve, because the only way to add a release is via createApplicationVersion, and that expects a uint64.
Tests are never bad, but I think these would be lower priority.